UnityPrefabs/Assets/Bullet.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Bullet : MonoBehaviour
+{
+
+    public float shootingForce = 1500f;
+    public Vector3 shootingDirection;
+    public GameObject explosionPrefab;
+
+    public float lifetime = 3f;
```

Comment:
variables for shooting bullet velocity magnitude, bullet unit vector, explosion prefab so direction can be assigned by player, and lifetime of bullets.